test/extensions/filters/http/ext_proc/http_client/ext_proc_http_integration_test.cc

```diff
@@ -40,7 +40,7 @@ using Http::LowerCaseString;
 struct ConfigOptions {
   bool downstream_filter = true;
   bool failure_mode_allow = false;
-  int64_t timeout = 900000000;
+  int64_t timeout = 10;
```

Comment:
can we change this variable to have units in it, e.g. timeout_ms or timeout_sec if the type is unit-less like int64_t? If it's an absl::Time then you wouldn't need it.

I am pretty confused about how this change from 900M to 10 makes any sense.

Please remember to do that always for everything :)

Reply:
Done for the naming.

There are two different timeouts in the ext_proc + HTTP. One is the ext_proc filter timer. The other is the HTTP client timer. The timeout handling code path are different.  If the response fails to arrive in time, whichever timeout setting is shorter fires first and shut down the external processing.  

There are existing test cases to trigger timeout events for each of them.  With this ext_proc filter timeout default setting change, we have to modify those tests accordingly to make them still work.